facial_recognition/app.py

```diff
@@ -1,7 +1,30 @@
 from flask import Flask
 
+from deepface import DeepFace
+
 app = Flask(__name__)
 
+
 @app.route("/")
 def hello_world():
-    return "<p>Hello, World!</p>"
\ No newline at end of file
+    return "<p>Team Wombat - Facial Recognition Service :) </p>"
+
+
+# TODO: Change to POST method and get the image as input
+# TODO: What if deepface fails ? - Handle error case
+# TODO: Any specific response format? - Status code, messages etc.
```

Comment:
Two choices here:
Use the HTTP status code to indicate the result. So 200 OK means good to go, 401 / 403 means unauthorized. OR use HTTP status codes for the request status itself (200 OK means I got your message and was able to run without error, and error codes mean I was not). Then you could put the result of the comparison into the body of the message using JSON